CS/jdbc/com/vehicledekho/jdbc/Connection.java

```diff
@@ -0,0 +1,32 @@
+package com.vehicledekho.jdbc;
+
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class Connection {
```

Comment:
Try to use the name that not conflict with Connection class in sql package like DBConnection.